boards/arm/degu_evk/Kconfig.defconfig

```diff
@@ -0,0 +1,103 @@
+# Kconfig - Degu Evaluation Kit configuration
+#
+# Copyright (c) 2019 Atmark Techno, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_DEGU_EVK
+
+config BOARD
+	default "degu_evk"
+
+if USB
+
+config USB_NRF52840
+	def_bool y
```

Comment:
Why do you use `def_bool`, not just `default`, here and a couple of times more below? These options are already defined as having the `bool` type, you only need to set default values for them.